### PR TITLE
Fix ofx gui events propagation

### PR DESCRIPTION
**Pull request by roymacdonald**
fixed `ofxGuiGroup` events propagation.
When using an `ofEasyCam` and an `ofxPanel` or `ofxGuiGroup` object,if moving the camera by dragging the mouse and the mouse was released over the gui, then when trying to change something in the gui would also change the `ofEasyCam` object. 
This issue got reported [here](https://forum.openframeworks.cc/t/clicks-on-ofxgui-panel-params-pass-through-to-main-window/31368/3) 

Additionally, I changed all the instances of `ofPoint` and `ofVec*f` into glm.

### CODE REVIEW

**Comment by arturoc** [comment]
can you separate this in two PR's is really difficult to see what belongs to solving the issue and what to porting to glm. there's even some color setting that doesn't seem related to either of those.

**Comment by roymacdonald** [comment]
@arturoc sure, but that's why I made 2 commits in this PR. Just thought it would be easier to review. If    you look at the commits you can see what's done in each case.
The color thing was just that ofxPanel was not drawing it's background. I should have added that in a different PR.
I will update this PR asap.
cheers

**Comment by roymacdonald** [comment]
ok. so I made this into a single commit, only with the changes made for the gui events propagation

**Comment by roymacdonald** [comment]
This is the other PR https://github.com/openframeworks/openFrameworks/pull/6207

**Comment by roymacdonald** [comment]
Hi, the CI gave some errrors which seem to be unrelated to my PR. Can these be re run? 
cheers

File: addons/ofxGui/src/ofxGuiGroup.cpp [hunk 1]
```diff
@@ -233,8 +233,8 @@ void ofxGuiGroup::add(ofParameter <ofFloatColor> & parameter){
 void ofxGuiGroup::clear(){
 	collection.clear();
 	parameters.clear();
-	
-	b.height = (bHeaderEnabled?headerRect.height:0) + spacing + spacingNextElement;
+//	there is no need for the following line as sizeChangedCB will handle it anyways, and change b.height.
```
Comment (arturoc) [inline addons/ofxGui/src/ofxGuiGroup.cpp]: can you just remove this comment?

Comment (roymacdonald) [reply addons/ofxGui/src/ofxGuiGroup.cpp]: Removed.
cheers

**Comment by arturoc** [comment]
thanks!

**Pull request merged**